src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
Using a List to represent the storage might not be the most efficient data structure, especially if you need to frequently search for a particular Fruit. Consider using a Map<Fruit, Integer> to keep track of the quantity of each Fruit, which could be more suitable for this purpose.